libr/core/canal.c

```diff
@@ -4909,7 +4944,7 @@ R_API void r_core_anal_esil(RCore *core, const char *str, const char *target) {
 		}
 		// if (op.type & 0x80000000 || op.type == 0) {
 		if (op.type == R_ANAL_OP_TYPE_ILL || op.type == R_ANAL_OP_TYPE_UNK) {
-			// i +=2;
+			i += mininstrsz - 1;
```

Comment:
@pancake Where do these changes come from? They do not seem to have anything to do with the anal block refactoring.

Reply:
to avoid infinite loops basically